tests/conftest.py

```diff
@@ -14,6 +15,11 @@
     _speedups = None  # type: ignore
 
 
+def pytest_report_header() -> list[str]:
+    """Return a list of strings to be displayed in the header of the report."""
+    return [f"Free-threaded: {bool(sysconfig.get_config_var('Py_GIL_DISABLED'))}"]
```

Comment:
Could [`sys._is_gil_disabled()`][_is_gil_disabled] be of more interest here?

As I understand it `Py_GIL_DISABLED` tells whether `--disable-gil` was set at python build time.
Even for `--disable-gil` builds the [PYTHONGIL] environment variable or loading of extension modules without a valid [Py_mod_gil] slot can cause the GIL to be enabled.  `_is_gil_disabled()` tells whether the GIL is actually in use or not.  (This seems more of interest when interpreting test results.)

[_is_gil_disabled]: https://docs.python.org/3.13/library/sys.html#sys._is_gil_enabled
[PYTHONGIL]: https://peps.python.org/pep-0703/#pythongil-environment-variable
[Py_mod_gil]: https://peps.python.org/pep-0703/#py-mod-gil-slot